model/event/event.go

```diff
@@ -74,6 +82,7 @@ func (e *EventLogEntry) SetBSON(raw mgobson.Raw) error {
 	}
 
 	// IDs for events were ObjectIDs previously, so we need to do this
+	// TODO (EVG-16650): Remove once old events are TTLed and/or migrated.
```

Comment:
Might make sense to split a new ticket to handle this work instead of keeping EVG-16650 open, since i imagine it'll be a few months

Reply:
done ([EVG-17214](https://jira.mongodb.org/browse/EVG-17214))